browser/brave_wallet/brave_wallet_service_factory.cc

```diff
@@ -55,7 +55,8 @@ KeyedService* BraveWalletServiceFactory::BuildServiceInstanceFor(
 
   return new BraveWalletService(
       shared_url_loader_factory, BraveWalletServiceDelegate::Create(context),
-      user_prefs::UserPrefs::Get(context), g_browser_process->local_state());
+      user_prefs::UserPrefs::Get(context), g_browser_process->local_state(),
+      context->GetPath());
```

Comment:
Let's move this to BraveWalletServiceDelegate

Reply:
Done